commcare/src/main/resources/webapp/partials/settings.html

```diff
@@ -28,10 +28,15 @@
              <a ng-click="addConfig()" type="button" class="form-group btn btn-success">
                  <i class="fa fa-plus"></i> {{msg('commcare.addConfig')}}
              </a>
+             <br />
```

Comment:
It would be much better if you could style it with .css instead of using `<br/>`